benchmarks/perf-tool/okpt/test/steps/steps.py

```diff
@@ -472,7 +475,7 @@ def _action(self):
         return results
 
     def _get_measures(self) -> List[str]:
-        measures = ['took', 'memory_kb']
+        measures = ['took', 'memory_kb', 'client_time']
```

Comment:
Could be nice to have a unit as well. `client_time_millis` or `client_time_seconds`

Reply:
Also, can we use constant?

Reply:
For this I don't see constant will make a lot of sense. Plus these are benchmark scripts, so I don't see a big advantage here.

For time unit, we don't do it for other metrics. Hence I feel for consistency we should keep it. Everything is already in milliseconds only.

Reply:
As long as its the same unit as took, thats fine

Reply:
its the same unit.